ThingIFSDK/ThingIFSDK/ThingIFAPI+OnBoard.swift

```diff
@@ -31,10 +31,14 @@ extension ThingIFAPI {
             // generate header
             var requestHeaderDict:Dictionary<String, String> = ["authorization": "Bearer \(owner.accessToken)"]
             
+            let vendorThingID: String
             if byVendorThingID {
+                vendorThingID = IDString
                 requestBodyDict.setObject(IDString, forKey: "vendorThingID")
                 requestHeaderDict["Content-type"] = "application/vnd.kii.OnboardingWithVendorThingIDByOwner+json"
             }else {
+                // TODO: Shall we input nil, but no way.
```

Comment:
まず、vendorThingIDの決定をする場所を今の場所はやめて、StandaloneThingを作成する直前にしましょう。
その上で、以下のようなコメントを追加してください。

```
Idealy server should send vendorThingID as response of onboarding,
and SDK should use the received vendorThingID.
However, current server implementation does not send vendorThingID.
So we used IDString if it is vendorThingID, otherwise we set it empty string.
This behavior should be fixed after server fixed.
```